_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -15,5 +15,6 @@ This section will be updated prior to the next release.
 * Ubuntu Xenial (16.04) is no longer supported. Users should upgrade to Bionic (18.04) prior to upgrading Foreman to this release.
 * Debian Stretch (9) is no longer supported. Users should upgrade to Buster (10) prior to upgrading Foreman to this release.
 * Smart Variables have been removed. Users should use Smart Class Parameters or Global Parameters instead. [Further details](https://community.theforeman.org/t/dropping-smart-variables/16176)
+* Cockpit single sign-on via "Web Console" button. For more information how to set it up, head over to [Remote Execution](/plugins/foreman_remote_execution/1.7/index.html#3.6CockpitIntegration) manual.
```

Comment:
Can you move this up to the Headline Features section above please? Also, looks like you forgot to add the change in the 1.7 rex manual